lib/src/scss-shared/_nx-variables.scss

```diff
@@ -47,9 +47,9 @@ $nx-font-size-heading-level-4: 14px;
 $nx-line-height: 1.5;
 
 // Form variables
-$nx-form-element-height: 32px;
-$nx-form-element-width-normal: 219px;
-$nx-form-element-width-wide: 395px;
+$nx-form-element-height: 40px;
```

Comment:
Now that buttons are 42px tall, should this be 42px?

Reply:
> Now that buttons are 42px tall, should this be 42px?

Now that layout is padding + font it's 42px: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/9e5655f8bd6cf083b80038a89ff85fa94fa8941d